web/src/components/tasks/cigarette-license/CigaretteLicense.tsx

```diff
@@ -79,6 +80,38 @@ export const CigaretteLicense = (props: Props): ReactElement => {
     });
   };
 
+  const doesStepHaveError = (step: number): boolean => {
```

Comment:
[dust] Within this function `getInvalidFieldIds()` is called twice, you could set it to a const at the top so it doesn't have to be called twice, just a thought.